From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Tue, 15 Jul 2014 18:28:10 +0200
Subject: units/serial-getty@.service: use the default RestartSec

For pluggable ttys such as USB serial devices, the getty is restarted
and exits in a loop until the remove event reaches systemd. Under
certain circumstances the restart loop can overload the system in a
way that prevents the remove event from reaching systemd for a long
time (e.g. at least several minutes on a small embedded system).

Use the default RestartSec to prevent the restart loop from
overloading the system. Serial gettys are interactive units, so
waiting an extra 100ms really doesn't make a difference anyways
compared to the time it takes the user to log in.

(cherry picked from commit 4bf04322b8b7ecca4f3d65cfc642d0ac16356129)
(cherry picked from commit 89b958ef6a43400d1f7b25c58b02f4b5c3c59b36)
---
 units/serial-getty@.service.m4 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/units/serial-getty@.service.m4 b/units/serial-getty@.service.m4
index aea078f..1c2a521 100644
--- a/units/serial-getty@.service.m4
+++ b/units/serial-getty@.service.m4
@@ -25,7 +25,6 @@ IgnoreOnIsolate=yes
 ExecStart=-/sbin/agetty --keep-baud %I 115200,38400,9600
 Type=idle
 Restart=always
-RestartSec=0
 UtmpIdentifier=%I
 TTYPath=/dev/%I
 TTYReset=yes
